### ISSUE TITLE
Forum : liste de diffusion HS ?

### ISSUE THREAD

**Issue by deuzeffe** [osm-fr/infrastructure#515]
Bonjour,

Depuis le 07/12, le mode "liste de diffusion" du forum (=envoi de chaque publication dans le forum par mail) ne fonctionne plus.

Merci de ce que vous pourrez faire.

**Comment by jocelynj** [comment]
Possible que ce soit lié à la suppression de la boîte email par gandi. Je regarde si on peut configurer le forum pour utiliser le smtp déjà disponible.

cc @cquest

**Comment by jocelynj** [comment]
Le smtp est configuré, et a l'air de marcher.

Par contre, on avait un pop3 configuré pour les réponses par email, et on n'a plus de pop3 maintenant. Peut-être qu'on devrait suivre https://meta.discourse.org/t/configure-direct-delivery-incoming-email-for-self-hosted-sites/49487,  avec un passage par mail.openstreetmap.fr pour enlever le spam ?

**Comment by deuzeffe** [comment]
@jocelynj je confirme que la partie smtp est tombée en marche, je viens de commencer à recevoir les mails. Merci à toi !
Pour ce qui est du POP3, je ne l'utilise pas. Aucune idée si ça va manquer à d'autres.

**Comment by vinber** [comment]
> Peut-être qu'on devrait suivre https://meta.discourse.org/t/configure-direct-delivery-incoming-email-for-self-hosted-sites/49487, avec un passage par mail.openstreetmap.fr pour enlever le spam ?

Est-ce en place @jocelynj ? pour info, sans pressin :D

**Comment by jocelynj** [comment]
Non, rien de mis en place pour le POP3, surtout parce que je n'ai pas vu de demande pour pouvoir poster sur le forum via email plutôt que via le site web.
